docs/styleguides/STYLEGUIDE.md

```
@@ -216,6 +216,28 @@ Use lists for a group of related items, such as a complete set of configuration
 - Use unordered (bulleted) lists when the order of the items is not important.
 - Use ordered (numbered) lists when giving steps or instructions to be followed in sequence.
 
+### Do not use emdashes in lists
+
+To not use emdashes in lists, if you want to separate the concise summary from the description of a list item, use bold text and colons instead.
+
+> ❌
+>
+> - **Memory** — Memory allows your agent to do stuff.
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)